### ISSUE TITLE
Forced quiet logs in CI mode -- not sure if this is a good idea

### ISSUE THREAD

**Issue by shaunhall** [nuxt/nuxt.js#9581]
The [documentation](https://nuxtjs.org/docs/2.x/configuration-glossary/configuration-build#quiet) for quiet mode states

> Default: Enabled when a CI or test environment is detected by std-env

Code is here
https://github.com/nuxt/nuxt.js/blob/8e725d34b3ffbbf5b4a94e1b86f45e6ffcc61f3d/packages/config/src/config/build.js#L4

### Actual Behaviour
You can't turn off quiet mode in a CI environment (without unsetting the CI environment variables, which might break other things)

### Expected Behaviour
Should not be forced to have quiet mode in a CI environment or the documentation should say that quiet mode is forced, not just default. Personally I would prefer quiet mode to be inferred from the flag only. 

Would be interested to hear other people's thoughts on this.

**Comment by manniL** [comment]
Hey 👋 
You should be able to override the `quiet` default by setting `build.quiet` in your `nuxt.config.js`. The linked config is only a list of sensible defaults ☺️

**Comment by shaunhall** [comment]
Hi @manniL. Thanks very much, that worked as you suggested :smiley:

**Comment by sveinugu** [comment]
Just wanted to provide my thoughts on this: Forced quiet logs in CI mode is NOT a good idea!

After a couple of hours of not understanding why my script that checks the output of a "npm run generate" command works differently on a GitHub actions Ubuntu container than on plain Ubuntu, I finally found that the issue probably was the lack of logging from the command itself and not my script. I have now spent 4-5 hours trying to figure out which part of a GitHub Actions workflow quiets the log. After going through logging in GitHub Actions itself, the node_setup action, npm configs and whatever else I could figure out, I finally found this issue!

The reason I did not check Nuxt output first is that I checked that clean installation of my Nuxt project was logging fully from the command line and I assumed the issue was due to a change in the environment and that nobody would add any context checking to the command line executable itself. However, I did at one point search the Nuxt web site with a number of terms, including "logging", without help. Finally, I found the nuxt CLI option "--quiet" and tried figuring out if there was an environment variable to set this and who would do such a thing! 

Basically, when debugging I believe it is common to assume that a config file + default values of a tool provides stable functionality across contexts. Especially in the context of a CI environment, where there are many moving parts, I think such an assumption is important to keep!
